<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>bluez.git - bluez</title>
<meta name='generator' content='cgit v0.9.0.3'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/bluez.git/atom/0003-fakehid-Use-the-same-constant-as-declared.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='bluez.git' href='/cgit/bluez.git/'>bluez.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>bluez</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/bluez.git/'>summary</a><a href='/cgit/bluez.git/refs/'>refs</a><a href='/cgit/bluez.git/log/0003-fakehid-Use-the-same-constant-as-declared.patch'>log</a><a class='active' href='/cgit/bluez.git/tree/0003-fakehid-Use-the-same-constant-as-declared.patch'>tree</a><a href='/cgit/bluez.git/commit/0003-fakehid-Use-the-same-constant-as-declared.patch'>commit</a><a href='/cgit/bluez.git/diff/0003-fakehid-Use-the-same-constant-as-declared.patch'>diff</a><a href='/cgit/bluez.git/stats/0003-fakehid-Use-the-same-constant-as-declared.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/bluez.git/log/0003-fakehid-Use-the-same-constant-as-declared.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/bluez.git/tree/'>root</a>/<a href='/cgit/bluez.git/tree/0003-fakehid-Use-the-same-constant-as-declared.patch'>0003-fakehid-Use-the-same-constant-as-declared.patch</a></div><div class='content'>blob: b6adf77b000a327d1fdc600e54cd07e6c4389ab6 (<a href='/cgit/bluez.git/plain/0003-fakehid-Use-the-same-constant-as-declared.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a class='no' id='n1' name='n1' href='#n1'>1</a>
<a class='no' id='n2' name='n2' href='#n2'>2</a>
<a class='no' id='n3' name='n3' href='#n3'>3</a>
<a class='no' id='n4' name='n4' href='#n4'>4</a>
<a class='no' id='n5' name='n5' href='#n5'>5</a>
<a class='no' id='n6' name='n6' href='#n6'>6</a>
<a class='no' id='n7' name='n7' href='#n7'>7</a>
<a class='no' id='n8' name='n8' href='#n8'>8</a>
<a class='no' id='n9' name='n9' href='#n9'>9</a>
<a class='no' id='n10' name='n10' href='#n10'>10</a>
<a class='no' id='n11' name='n11' href='#n11'>11</a>
<a class='no' id='n12' name='n12' href='#n12'>12</a>
<a class='no' id='n13' name='n13' href='#n13'>13</a>
<a class='no' id='n14' name='n14' href='#n14'>14</a>
<a class='no' id='n15' name='n15' href='#n15'>15</a>
<a class='no' id='n16' name='n16' href='#n16'>16</a>
<a class='no' id='n17' name='n17' href='#n17'>17</a>
<a class='no' id='n18' name='n18' href='#n18'>18</a>
<a class='no' id='n19' name='n19' href='#n19'>19</a>
<a class='no' id='n20' name='n20' href='#n20'>20</a>
<a class='no' id='n21' name='n21' href='#n21'>21</a>
<a class='no' id='n22' name='n22' href='#n22'>22</a>
<a class='no' id='n23' name='n23' href='#n23'>23</a>
<a class='no' id='n24' name='n24' href='#n24'>24</a>
<a class='no' id='n25' name='n25' href='#n25'>25</a>
<a class='no' id='n26' name='n26' href='#n26'>26</a>
</pre></td>
<td class='lines'><pre><code>From a354165e58f937ee12c16ab48ce334b664c8f163 Mon Sep 17 00:00:00 2001
From: Bastien Nocera &lt;hadess@hadess.net&gt;
Date: Sun, 17 Jun 2012 01:29:01 +0200
Subject: [PATCH 3/3] fakehid: Use the same constant as declared

ps3remote_keymap[] uses 0xff as the max value, so should we.
---
 input/fakehid.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/input/fakehid.c b/input/fakehid.c
index a125356..a758413 100644
--- a/input/fakehid.c
+++ b/input/fakehid.c
@@ -335,7 +335,7 @@ static int ps3remote_setup_uinput(struct fake_input *fake,
 	}
 
 	/* enabling keys */
-	for (i = 0; i &lt; 256; i++)
+	for (i = 0; i &lt; 0xff; i++)
 		if (ps3remote_keymap[i] != KEY_RESERVED)
 			if (ioctl(fake-&gt;uinput, UI_SET_KEYBIT,
 						ps3remote_keymap[i]) &lt; 0) {
-- 
1.7.10

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.9.0.3 at 2012-11-07 00:56:04 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
